docs/pages/example_workflows/static_manipulation/step_5_evaluation.rst

```diff
@@ -138,7 +138,7 @@ than the single environment evaluation because of the parallel evaluation.
 
 .. code-block:: text
 
-   Metrics: {'success_rate': 0.605, 'revolute_joint_moved_rate': 0.955, 'num_episodes': 200}
+   Metrics: {'success_rate': 0.605, 'door_moved_rate': 0.955, 'num_episodes': 200}
```

Comment:
I think that you didn't intend to change this no?

I think what is happening here is that you need to rebase your MR on the top of main. There will likely be some conflicts that you'll need to sort out.